cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go

```diff
@@ -1930,4 +1930,34 @@ var InstanceTypes = map[string]*InstanceType{
 		MemoryMb:     32768,
 		GPU:          0,
 	},
+	"r6g.12xlarge": {
+		InstanceType: "r6g.12xlarge",
+		VCPU:         48,
+		MemoryMb:     393216,
+		GPU:          0,
+	},
+	"r6g.16xlarge": {
+		InstanceType: "r6g.16xlarge",
+		VCPU:         64,
+		MemoryMb:     524288,
+		GPU:          0,
+	},
+	"r6g.24xlarge": {
```

Comment:
Where is this `r6g.24xlarge` information coming from, as I can't seem to find any information on it, and run `make generate doesn't seem to pull this information in for me.